src/SDL_assert.c

```diff
@@ -245,7 +245,9 @@ static SDL_AssertState SDLCALL SDL_PromptAssertion(const SDL_AssertData *data, v
             state = (SDL_AssertState)selected;
         }
     } else {
-#ifdef SDL_PLATFORM_EMSCRIPTEN
+#ifdef SDL_PLATFORM_PRIVATE_ASSERT
+        SDL_PRIVATE_PROMPTASSERTION()
```

Comment:
```suggestion
        SDL_PRIVATE_PROMPTASSERTION();
```

Reply:
This is usually implemented something like:
`#define SDL_PRIVATE_PROMPTASSERTION() SDL_PrivateAssertionImpl()`
or, for inline code, something like:
```c
#define SDL_PRIVATE_PROMPTASSERTION() \
do { \
   ... \
} while( 0 )
```